chain/index/api.go

```diff
@@ -200,14 +206,30 @@ func (si *SqliteIndexer) verifyIndexedData(ctx context.Context, ts *types.TipSet
 		return xerrors.Errorf("failed to get next tipset for height %d: %w", ts.Height(), err)
 	}
 
+	// if non-reverted events exist which means that tipset `ts` has been executed, there should be 0 reverted events in the DB
```

Comment:
@aarshkshah1992 I have moved this part of top of the function, because if we have reverted events in the tipset then we can just return without loading all the messages.